cfg/config.go

```diff
@@ -196,6 +196,8 @@ type MetadataCacheConfig struct {
 
 	ExperimentalMetadataPrefetchOnMount string `yaml:"experimental-metadata-prefetch-on-mount"`
 
+	NegativeTtlSecs int64 `yaml:"negative-ttl-secs"`
```

Comment:
Could you add a integration test for testing negative-ttl-secs.  You could maybe have a test that the 2nd read after Write expects a FileNotFound error with high negative-ttl enabled.

Reply:
yes I am planning to add 3 basic cases - ttl: 0, ttl -1 and default value in the subsequent PR.